### PR TITLE
Adds two new research problems:  Waku incentivization and building applications on top of waku  

### PR DESCRIPTION
**Pull request by staheri14**
In this PR, I elaborated on two of my suggested research problems.

### CODE REVIEW

File: open-problems.md [hunk 1]
```diff
@@ -20,14 +20,37 @@ A new version of discv5 that is both efficient and provides eclipse mitigation i
...
 ### Privacy-Preserving Incentivizations of waku protocols
 
-* Incentivise relay
+Waku protocols are designed in such a way that resource-limited devices can make up for their lack of resources by relying on the more powerful peers in the network.
+This can be seen at all the layers of the Waku protocol stack.
+One of which is the Waku relay protocol in which peers voluntarily join the routing layer of Waku and aid message propagation.
+While contributing peers may have altruistic reasons, or be a stakeholder in some higher-level applications and would like to deliver high-quality service via reliable infrastructure.
+However, this cannot be generalized to all peers hence peers need to be incentivized for their participation and contribution in the network regardless of being tied to any specific application, or necessarily having altruistic wills.
+
+One of the most challenging protocols of Waku for incentivization is the Waku Relay protocol.
```
Comment (jm-clius) [inline open-problems.md:29]:
```suggestion
One of the most challenging Waku protocols for incentivization is the Waku Relay protocol.
```

File: open-problems.md [hunk 2]
```diff
@@ -20,14 +20,37 @@ A new version of discv5 that is both efficient and provides eclipse mitigation i
...
 
-examples comprise
 
-* p2p social network
-* evoting system
+### Devising novel applications of the Waku protocol stack
+
+Waku provides a suite of modular p2p protocols to enable the realization of web2 applications in the decentralized web3 world in the form of DApps. 
+Waku protocol stack features all the necessary components for the DApp developers namely, the networking/routing layer, and multiple discovery methods.
+It also provides various request-reply protocols to allow resource provisioning for resource-limited devices.
```
Comment (jm-clius) [inline open-problems.md:42]:
```suggestion
It also provides various request-response protocols to allow resource provisioning for resource-limited devices. 
```

File: open-problems.md [hunk 3]
```diff
@@ -20,14 +20,37 @@ A new version of discv5 that is both efficient and provides eclipse mitigation i
...
-examples comprise
 
-* p2p social network
-* evoting system
+### Devising novel applications of the Waku protocol stack
+
+Waku provides a suite of modular p2p protocols to enable the realization of web2 applications in the decentralized web3 world in the form of DApps. 
+Waku protocol stack features all the necessary components for the DApp developers namely, the networking/routing layer, and multiple discovery methods.
+It also provides various request-reply protocols to allow resource provisioning for resource-limited devices. 
+For example, store protocol (to account for storage limitations), and filter and light push protocols (to aid bandwidth-constrained peers).
```
Comment (jm-clius) [inline open-problems.md:43]:
```suggestion
For example, store protocol (to account for storage limitations), and filter and lightpush protocols (to aid bandwidth-constrained peers).
```

**Review by jm-clius — COMMENTED** [review]
Great research problems. I have added some comments re phrasing and clarity!

**Comment by staheri14** [comment]
@jm-clius thanks, addressed all the comments https://github.com/vacp2p/vac.dev/pull/93/commits/71d0092bd5eb0ee138ff69e8ad535ed8cbc0013b

**Comment by oskarth** [comment]
Thanks! Make sure these gets added to master branch, right now it is just in staging

**Pull request merged**